### PR TITLE
Make timeout run conditional instead of allowing 0

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes: #2669 

Builds: https://dev.azure.com/Betaflight/Betaflight%20Nightlies/_build/results?buildId=4660&view=artifacts&pathAsName=false&type=publishedArtifacts

### CODE REVIEW

File: src/js/main.js
Comment (Asizon) [inline src/js/main.js:405]: @haslinghuis How we can know here if it need a timeout?

Comment (Asizon) [reply src/js/main.js:405]: Ok, it is aplied on pid tuning tab for now

Comment (haslinghuis) [reply src/js/main.js:405]: Timeout is declared on line 253 as 0. Assigned to 100 on line 258 if needed. So the timeout is now only applied when it's set.

**Review by Asizon — DISMISSED** [review]

File: src/js/main.js
Comment (Asizon) [reply src/js/main.js:405]: Ok, then maybe it should be if(timeout > 0)??

Comment (haslinghuis) [reply src/js/main.js:405]: Needed a bigger timeout 😕 

if (timeout) should be the same as if (timeout > 0)

Comment (Asizon) [reply src/js/main.js:405]: No, because timeout is always defined at 0,there is always timeout defined.

Comment (haslinghuis) [reply src/js/main.js:405]: Just tested, it takes the else clause when timeout === 0

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by limonspb — DISMISSED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Comment by ctzsnooze** [comment]
tested on MacBook Pro with F405, 500 works great.  
Otherwise with current master I get garbage characters in the CLI after moving from a slider tab to CLI.

**Review by Asizon — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Pull request merged**
